Sources/Protocols/CellItem/CollectionViewCellItem.swift

```diff
@@ -83,6 +83,20 @@ extension CollectionViewSiblingCellItem {
             objc_setAssociatedObject(self, &AssociatedKeys.sectionItem, newValue, .OBJC_ASSOCIATION_ASSIGN)
         }
     }
+    
+    public weak var cell: UICollectionViewCell? {
+        get {
```

Comment:
Please add docs for new public API

Reply:
other properties there have not any docs

Reply:
And this isn't good. There are 2 options:
1. You can start to fix this by writing docs only for this property
2. You can fix this by writing docs for **all** properties